tests/videostore.test.js

```diff
@@ -74,7 +74,7 @@ describe('Video Store:', function () {
     browser.getText('h3', 'Sign in')
   })
 
-  it('should show an error if the user does not have enough PTI @watch', function () {
```

Comment:
I am pretty sure that this (and the `@watch` flag below) were not intended to remain in the codebase

Reply:
you are correct :-)